runbot_build_instructions/runbot_build.py

```diff
@@ -169,3 +170,13 @@ def cmd(self, cr, uid, ids, context=None):
             "--workers=0",
         ] + params
         return cmd, mods
+
+    @api.cr_uid_ids_context
+    def server(self, cr, uid, ids, *l, **kw):
```

Comment:
I used the old API mainly for consistency. The rest of the file is still completely in the old API.